.docker/django/uwsgi_configuration.ini

```diff
@@ -33,3 +33,7 @@ worker-reload-mercy = 60  # How long to wait before forcefully killing workers (
 ignore-sigpipe = true
 ignore-write-errors = true
 disable-write-exception = true
+
+# Exclude health and readiness probes from logs
+route = ^/readiness$ donotlog:
+route = ^/healthz$ donotlog:
```

Comment:
Nice

Reply:
Can't take credit for this 😔 copied from here: https://github.com/City-of-Helsinki/kukkuu/blob/master/.prod/uwsgi.ini#L12-L13